index.html

```diff
@@ -73,6 +76,198 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
           </ul>
         </div>
       </section>
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/198854940-7ba12ccc-8bf2-472f-8ec8-771dccb5fbe7.PNG)

**[Optional]**

- [ ] Your background is not expanding on the whole screen as you can see. Kindly fix it to make your design look similar to the Figma template 😊to make your design look professional 👍

Reply:
Thanks i will #Hammas-saleem